analyzer.py

```diff
@@ -0,0 +1,52 @@
+import pickle
+import twitter
+import random
+
+markov = dict()
+
+def generate_markov(messages,num=2):
+    '''
+    Process the input message and generate a markov map for the prefix and surfix.
+    '''
+    global markov
+    for i in range(len(messages)):
+        message = messages[i]
+        current_words = tuple()
+        for word in message.rstrip().split():
+            word = word.strip('@')
+            word = word.strip('#')
+            word = word.lower()
+            if len(current_words)<num:
+                current_words= current_words + (word,)
+            else:
+                if markov.get(current_words,0) == 0:
+                    markov[current_words] = [word]
+                else:
+                    markov[current_words].append(word)
+                current_words = current_words[1:]+ (word,)
+
+def generate_random(a,b):
```

Comment:
👏 Very cool implementation of markov stuff, could there be a way to integrate # and @, considering the importance of that in tweets?